jerry-core/ecma/builtin-objects/ecma-builtin-array-prototype.c

```diff
@@ -1218,10 +1079,12 @@ ecma_builtin_array_prototype_object_sort (ecma_value_t this_arg, /**< this argum
   /* Sorting. */
   if (copied_num > 1 && ecma_is_value_empty (ret_value))
   {
+    const sort_compare_cb sortfn = &ecma_builtin_array_prototype_object_sort_compare_helper;
```

Comment:
I might wasn't clear. Ok, here is an example from jerry-core.h:

```
/**
 * Function type for allocating buffer for JerryScript instance.
 */
typedef void *(*jerry_instance_alloc_t) (size_t size, void *cb_data_p);

/**
 * Type information of a native pointer.
 */
typedef struct
{
  jerry_object_native_free_callback_t free_cb; /**< the free callback of the native pointer */
} jerry_object_native_info_t;
```

So we put `_t` after types, including callbacks. However, for the names of the variables we put `_cb` for callbacks, `_p` for pointers. This was the rule way before I joined to the project. So the correct form looks like this:

`const sort_compare_t sort_cb = &ecma_builtin_array_prototype_object_sort_compare_helper;`

Reply:
I understand now. Made the change.